traits/trait_numeric.py

```diff
@@ -296,104 +283,83 @@ def _default_for_dtype_and_shape(self, dtype, shape):
         return value
 
 
-# -------------------------------------------------------------------------------
-#  'Array' trait:
-# -------------------------------------------------------------------------------
-
-
 class Array(AbstractArray):
-    """ Defines a trait whose value must be a numpy array.
+    """ A trait type whose value must be a Numpy array.
```

Comment:
Super-picky nitpick:

```suggestion
    """ A trait type whose value must be a NumPy array.
```